server.js

```diff
@@ -1,10 +1,8 @@
-var express = require('express')
```

Comment:
(Error) Line 1: Expected linebreaks to be 'LF' but found 'CRLF'. - linebreak-style